services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -353,6 +353,26 @@ public static Builder builder() {
   @Nullable
   public abstract String exclude();
 
+  /**
+   * Include certain road types from routing. The default is to not include anything from the
+   * profile selected. The following include flags are available for each profile:
```

Comment:
@ktatterso is this accurate? If not, could you share the docs?

Reply:
```suggestion
   * Include certain road types in routing. By default, none of the road types listed below are included. The following include flags are available for each profile:
```